org.openhab.binding.zigbee/src/main/java/org/openhab/binding/zigbee/handler/ZigBeeThingHandler.java

```diff
@@ -597,6 +624,11 @@ public void handleConfigurationUpdate(Map<String, Object> configurationParameter
 
         // Persist changes
         updateConfiguration(configuration);
+
+        if (initializeDevice) {
```

Comment:
Question: What is the difference for you between `initializeDevice` and `reinitializeDevice`? (If there is none, you could use one of the words both for the variable and the method.)

Reply:
I named the variable `initializeDevice` because it is related to the config param `zigbee_initialise_device`. The method name is `reinitializeDevice` because a re-initialization is what it does when the config param is set to `true`.
Do you think it's confusing or not good to understand?

Reply:
> Do you think it's confusing or not good to understand?

I think that it is ok, but not ideal.